source/server-discovery-and-monitoring/tests/rs/ls_timeout.json

```diff
@@ -95,6 +251,17 @@
           "b:27017": {
             "type": "RSSecondary",
             "setName": "rs"
+          },
+          "c:27017": {
+            "type": "RSOther",
+            "setName": "rs"
+          },
+          "d:27017": {
+            "type": "RSGhost"
```

Comment:
Is the omission of "setName" for RSGhosts important?

Reply:
Ghosts don't necessarily know what their set name is.  See https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/tests/rs/ghost_discovered.yml for another example.

Reply:
But I don't think it matters to this particular test one way or the other.

Reply:
Thanks